static/js/settings_users.js

```diff
@@ -114,14 +114,14 @@ function populate_users(realm_people_data) {
 
     var bots_table_html = "";
     _.each(bots, function (user) {
-        var bot_html = templates.render("admin_user_list", {user: user});
+        var bot_html = templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin});
         bots_table_html = bots_table_html.concat(bot_html);
     });
     bots_table.append(bots_table_html);
 
     _.each(active_users, function (user) {
         var activity_rendered;
-        var row = $(templates.render("admin_user_list", {user: user}));
+        var row = $(templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin}));
```

Comment:
If each row is for a different user, then why do they each use `page_params.is_admin`?